graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasElasticsearchQuery.java

```diff
@@ -144,6 +144,19 @@ private String perfromDirectIndexQuery(String query) throws IOException {
         return EntityUtils.toString(response.getEntity());
     }
 
+    private String performDirectIndexQueryWithSource(String query) throws IOException {
+        HttpEntity entity = new NStringEntity(query, ContentType.APPLICATION_JSON);
+
+        String endPoint = index + "/_search?_source=true";
```

Comment:
Why do we need _source=true?

and if we need it, no need to create a separate function with a new name. You can use method overloading and call one method from the other to avoid code duplication.

Reply:
Because the task information is in the source of the document in Elasticsearch.

Will updated shortly.

Reply:
Can you please check [latest commit](https://github.com/atlanhq/atlas-metastore/pull/727/commits/2c2e6d69ee30291063d80f1292b1aae7d2e0ff20).

Reply:
Discuss over call, Burak will make the changes